src/components/ThemesSection.tsx

```diff
@@ -17,40 +17,82 @@ import populationImg from "../public/themes/population.svg";
 import regionsImg from "../public/themes/regions.svg";
 import techImg from "../public/themes/tech.svg";
 import transportImg from "../public/themes/transport.svg";
+import serverConfig from "@/config/serverConfig";
 
-const allThemes = [
+export const allThemes = [
   {
     name: "Agriculture",
     img: agricultureImg,
     desc: "Agriculture, fisheries, forestry and food",
+    url: "publications.europa.eu/resource/authority/data-theme/AGRI",
+  },
+  {
+    name: "Economy",
+    img: economyImg,
+    desc: "Economy and finance",
+    url: "publications.europa.eu/resource/authority/data-theme/ECON",
   },
-  { name: "Economy", img: economyImg, desc: "Economy and finance" },
   {
     name: "Education",
     img: educationImg,
     desc: "Education, culture and sport",
+    url: "publications.europa.eu/resource/authority/data-theme/EDUC",
+  },
+  {
+    name: "Energy",
+    img: energyImg,
+    url: "publications.europa.eu/resource/authority/data-theme/ENER",
+  },
+  {
+    name: "Environment",
+    img: envImg,
+    url: "publications.europa.eu/resource/authority/data-theme/ENVI",
   },
-  { name: "Energy", img: energyImg },
-  { name: "Environment", img: envImg },
   {
     name: "Government",
     img: governmentImg,
     desc: "Government and public sector",
+    url: "publications.europa.eu/resource/authority/data-theme/GOVE",
+  },
+  {
+    name: "Health",
+    img: healthImg,
+    url: "publications.europa.eu/resource/authority/data-theme/HEAL",
   },
-  { name: "Health", img: healthImg },
   {
     name: "International Issues",
     img: internationalIssuesImg,
+    url: "publications.europa.eu/resource/authority/data-theme/INTR",
   },
   {
     name: "Justice",
     img: justiceImg,
     desc: "Justice, legal system and public safety",
+    url: "publications.europa.eu/resource/authority/data-theme/JUST",
```

Comment:
important: please add `https://` to the URL, it is part of the value.

In future releases we will fetch the values from the backend, which may break or add side effects in the frontend if we have any logic to handling the missing protocol.